Code Quality/Reviewing a pull request.md

```diff
@@ -0,0 +1,64 @@
+> Optimism is an occupational hazard of programming; feedback is the treatment. - Kent Beck
+
+This guide should help you review code better. A good code review will provide a lot of useful things, but some of them are:
+
+-   a sense of security (that the feature works well),
+-   a learning opportunity for both parties,
+-   architectural check,
+-   code quality check.
+
+## Language of the code review
+
+-   be concise
+-   do not ask, advise
+-   do not assume the level of knowledge
+-   be open for discussion, but do not bike ßshed
+-   recommend fixes instead of pointing out errors
+
+### Good examples:
+
+> this won't work because the function you're calling as `async`.
+
+> this file is palaced in the incorrect folder, it should go to `components/shared`
+
+> I'm not sure this does what you want it to do. Looking at the service as a whole you should probably allow the end user to construct this out of available functions.
+
+### Bad examples:
+
+> can you please update this function documentation as well?
+
+> this is not according to our guidelines
+
+> naming is not correct here
+
+## When do I review pull requests?
+
+Reviewing a pull request could (and should) take time. This is not to be rushed since you vouch for the quality of the code that's on review.
+
+If you review code for either multiple projects or you work in a larger team you might want to think about assigning a certain fixed part of the day to do them (e.g. I review pull requests in the morning). Everything that comes during the day is to be reviewed first thing tomorrow. This should not include urgent things that block hot fixes, etc.
```

Comment:
Also worth mentioning that it should be taken into account during estimates?

Reply:
HM - Good point!